nuxeo-runtime/nuxeo-runtime/src/main/java/org/nuxeo/runtime/model/impl/ExtensionPointImpl.java

```diff
@@ -102,7 +103,13 @@ public Extension createExtension(Element element) {
                     }
                 }
             }
-            contribs = xmap.loadAll(new XMapContext(extension.getContext()), extension.getElement());
+            try {
+                contribs = xmap.loadAll(new XMapContext(extension.getContext()), extension.getElement());
+            } catch (XMapException e) {
+                throw new RuntimeException(
+                        e.getMessage() + " while processing component: " + extension.getComponent().getName().getName(),
+                        e);
```

Comment:
I find `Cannot load class: this-is-not-a-class while processing component: OverridingXPoint-witherror` readable enough. Until we have an enforced global style guide for error messages I don't think it's worth debating too much.